get-started/search-ai-platform.md

```diff
@@ -0,0 +1,131 @@
+{{beats}} are available for many standard observability data scenarios, including audit data, log files and journals, cloud data, availability, metrics, network traffic, and Windows event logs.
+
+[Learn more about {{beats}}](beats://reference/index.md).
+
+### {{es}} ingest pipelines [stack-components-ingest-pipelines]
+
+Ingest pipelines let you perform common transformations on your data before indexing them into {{es}}.
+You can configure one or more "processor" tasks to run sequentially, making specific changes to your documents before storing them in {{es}}.
+
+[Learn more about ingest pipelines](/manage-data/ingest/transform-enrich/ingest-pipelines.md).
+
+### {{ls}} [stack-components-logstash]
+
+{{ls}} is a data collection engine with real-time pipelining capabilities.
+It can dynamically unify data from disparate sources and normalize the data into destinations of your choice.
+{{ls}} supports a broad array of input, filter, and output plugins, with many native codecs further simplifying the ingestion process.
+
+[Learn more about {{ls}}](logstash://reference/index.md).
+
+## Store, search, and analyze [_store]
+
+{{es}} is the distributed search, storage, and analytics engine at the heart of the {{search-platform}}.
+
+### {{es}} [stack-components-elasticsearch]
+
+{{es}} provides near real-time search and analytics for all types of data.
+Whether you have structured or unstructured text, numerical data, vectors, or geospatial data, {{es}} can efficiently store and index it in a way that supports fast searches.
+
+{{es}} is built to be a resilient and scalable distributed system.
+It runs as a cluster of one or more servers, called nodes.
+When you add data to an index, it's divided into pieces called shards, which are spread across the various nodes in the cluster.
+This architecture allows {{es}} to handle large volumes of data and ensures that your data remains available even if a node fails.
+
+Nearly every aspect of {{es}} can be configured and managed programmatically through its REST APIs.
+This allows you to automate repetitive tasks and integrate Elastic management into your existing operational workflows.
+For example, you can use the APIs to manage indices, update cluster settings, run complex queries, and configure security.
+This API-first approach is fundamental to enabling infrastructure-as-code practices and managing deployments at scale.
+
+Learn more about [the {{es}} data store](/manage-data/data-store.md), its [distributed architecture](/deploy-manage/distributed-architecture.md), and [APIs](elasticsearch://reference/elasticsearch/rest-apis/index.md).
+
+## Explore [_consume]
+
+Use {{kib}} to explore and visualize the data that's stored in {{es}} and to manage the {{search-platform}}.
+You can use the {{es}} clients to access data directly by using common programming languages.
+
+### {{kib}} [stack-components-kibana]
```

Comment:
I think here we need to present Kibana not only as a set of Analytics features but also as where all solutions (and their dedicated capabilities) come to life